nservicebus/throughput-tool/sql-transport.md

```diff
@@ -12,12 +12,20 @@ This article details how to collect endpoint and throughput data when the system
 
 ## Running the tool
 
-Once installed, execute the tool with the database connection string used by SQL Server endpoints, as in this example:
+Once installed, execute the tool with the database connection string used by SQL Server endpoints.
+
+If the tool was [installed as a .NET tool](/nservicebus/throughput-tool/#installation-net-tool-recommended):
 
 ```shell
 throughput-counter sqlserver --connectionString "Server=SERVER;Database=DATABASE;User=USERNAME;Password=PASSWORD;Trust Server Certificate=true;"
 ```
 
+Or, if using the [self-contained executable](/nservicebus/throughput-tool/#installation-self-contained-executable):
+
+```shell
+Particular.EndpointThroughputCounter.exe sqlserver --connectionString "Server=SERVER;Database=DATABASE;User=USERNAME;Password=PASSWORD;Trust Server Certificate=true;"
```

Comment:
I think you said we don't need the Trust Server Certificate=true. Should we remove that?

Reply:
Removed it. Also removed the no-longer-valid stuff about only being able to get a rowversion when the table is non-empty.